[2024-09-04T18:18:00.471+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-04T18:18:00.482+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag_v3.third_task manual__2024-09-04T18:17:57.686359+00:00 [queued]>
[2024-09-04T18:18:00.487+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag_v3.third_task manual__2024-09-04T18:17:57.686359+00:00 [queued]>
[2024-09-04T18:18:00.489+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 6
[2024-09-04T18:18:00.499+0000] {taskinstance.py:2879} INFO - Executing <Task(BashOperator): third_task> on 2024-09-04 18:17:57.686359+00:00
[2024-09-04T18:18:00.510+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=330) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-04T18:18:00.512+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag_v3', 'third_task', 'manual__2024-09-04T18:17:57.686359+00:00', '--job-id', '99', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpit364zlk']
[2024-09-04T18:18:00.523+0000] {standard_task_runner.py:72} INFO - Started process 331 to run task
[2024-09-04T18:18:00.527+0000] {standard_task_runner.py:105} INFO - Job 99: Subtask third_task
[2024-09-04T18:18:00.656+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag_v3.third_task manual__2024-09-04T18:17:57.686359+00:00 [running]> on host f1e60abe9381
[2024-09-04T18:18:00.781+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='abhay' AIRFLOW_CTX_DAG_ID='my_first_dag_v3' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2024-09-04T18:17:57.686359+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-04T18:17:57.686359+00:00'
[2024-09-04T18:18:00.785+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-04T18:18:00.807+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-04T18:18:00.813+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo Yo I am task3']
[2024-09-04T18:18:00.866+0000] {subprocess.py:86} INFO - Output:
[2024-09-04T18:18:00.868+0000] {subprocess.py:93} INFO - Yo I am task3
[2024-09-04T18:18:00.871+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-04T18:18:00.923+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-04T18:18:00.925+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=my_first_dag_v3, task_id=third_task, run_id=manual__2024-09-04T18:17:57.686359+00:00, execution_date=20240904T181757, start_date=20240904T181800, end_date=20240904T181800
[2024-09-04T18:18:00.953+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-09-04T18:18:00.973+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
